explore/src/main/scala/explore/config/SpectroscopyConfigurationPanel.scala

```diff
@@ -92,7 +105,16 @@ object SpectroscopyConfigurationPanel extends ConfigurationFormats:
             changeAuditor = ChangeAuditor.posInt.optional,
             disabled = p.readonly
           ).clearable(^.autoComplete.off),
-          SignalToNoiseAt(options, p.readonly, p.units, p.calibrationRole),
+          FormEnumDropdownView(
+            id = "exposureMode".refined,
+            value = emv,
+            label = ReactFragment(
+              "Exposure Mode",
+              HelpIcon("configuration/exposure-mode.md".refined)
+            ),
+            disabled = true
+          ),
+          snMode.asView.map(SignalToNoiseAt(_, p.readonly, p.units, p.calibrationRole)),
```

Comment:
Here we actually switch between the 2 modes but only show editing controls for S/N?

Reply:
Yes because this is the initial PR, the selector is disable, I have to add the time and count editor and make it common to both editors